examples/tutorials/tutorial_2D_crm.py

```diff
@@ -84,7 +81,7 @@
         reference_geometry=fl.ReferenceGeometry(
             moment_center=[0.25, 0.005, 0], moment_length=[1, 1, 1], area=0.01
```

Comment:
Same comment as in the tutorial_2D_30p30n.py file.

Reply:
Yes